core/templates/dev/head/pages/exploration_editor/simple_editor_tab/StatesToQuestionsService.js

```diff
@@ -174,6 +174,18 @@ oppia.factory('StatesToQuestionsService', [
       // current set of states to a list of questions.
       getQuestions: function() {
         return getQuestions();
+      },
+      getHumanReadableQuestionType: function(interactionId) {
+        var readableQuestionName;
+        SUPPORTED_INTERACTION_TYPES.forEach(function(interaction) {
+          if (interaction.id === interactionId) {
+            readableQuestionName = interaction.name;
+          }
+        });
+        if (!readableQuestionName) {
+          throw Error('Invalid Interaction ID');
```

Comment:
Doesn't look done to me; please double-check before marking something as "Done".